src/smt/smt_engine.cpp

```diff
@@ -81,6 +81,7 @@
 #include "proof/theory_proof.h"
 #include "proof/unsat_core.h"
 #include "prop/prop_engine.h"
+#include "smt/abduction_solver.h"
```

Comment:
do you still need to include `sygus_abduct.h` (few rows down)?

Reply:
No, ill remove.  (There are many in this list we don't need actually) :)